lib/node_modules/@stdlib/array/empty-like/docs/types/index.d.ts

```diff
@@ -399,8 +399,8 @@ declare function emptyLike( x: Float32Array, dtype?: DataType ): Float32Array;
 * @example
 * var zeros = require( `@stdlib/array/zeros` );
 *
-* var x = zeros( 2, 'float32' );
-* // returns <Float32Array>[ 0.0, 0.0 ]
+* var x = zeros( 2, 'complex128' );
+* // returns <Complex128Array>[ 0.0, 0.0 ]
```

Comment:
```suggestion
* // returns <Complex128Array>
```

Complex number arrays are harder to document in returns annotations, as a 2-element complex array has a length 4 buffer. This is why we typically use `real` and `imag` in examples involving complex number arrays, rather than showing the buffer contents.

In this case, it is enough to simply document that a complex number array is returned, without showing buffer contents.